docs/content/en/integration/openid-connect/roundcube/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "Roundcube"
+description: "Integrating Roundcube and Dovecot with the Authelia OpenID Connect 1.0 Provider."
+lead: ""
+date: 2023-10-23T15:16:47+02:00
+draft: true
+images: []
+menu:
+  integration:
+    parent: "openid-connect"
+weight: 620
+toc: true
+community: true
+---
+
+## Tested Versions
+
+All packages are from Debian 12.
+
+* [Authelia]
+  * [4.37.5](https://github.com/authelia/authelia/releases/tag/v4.37.5)
```

Comment:
```suggestion
  * [4.38.0](https://github.com/authelia/authelia/releases/tag/v4.38.0)
```

We're only accepting contributed docs for the upcoming 4.38.0 which will be the basis for this change and some changes to the config example.

Reply:
OK!

Reply:
BTW, is there a 4.38-0 build available for Debian that I can test? You never know...

Reply:
https://buildkite.com/authelia/authelia/builds/24994#_ artifacts should have a relevant deb.